content/posts/scikit-learn/gpu-ci/index.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Automated tests with GPUs for your project"
+date: 2024-07-19T15:24:17+01:00
+draft: false
+description: "Setting up CI with a GPU to test your code"
+tags: ["scikit-learn", "ci", "gpu", "cuda"]
+displayInList: true
+author: ["Tim Head <>timhead"]
```

Comment:
```suggestion
author: ["Tim Head <>timhead"]
```

Typo?

Reply:
Typo

Reply:
Err, still wrong... my suggestion was identical to the original :upside_down_face: ... should be `Tim Head <timhead>`.